ddtrace/internal/gitmetadata.py

```diff
@@ -0,0 +1,112 @@
+import os
+import typing
+
+from ddtrace.internal.utils import formats
+
+
+__GITMETADATA_TAGS = None  # type: typing.Optional[typing.Dict[str, str]]
```

Comment:
Is there a reason why we are using double underscore prefixes for private variables here?

Reply:
When I studied python (long long time ago) it means that variable is private and nobody outside the module have access (except some tricks), is something changed?

Reply:
A single underscore is generally enough. A double underscore prefix (with no double underscore suffix) is generally used in class methods and attributes and causes the variable names to be mangled. We very rarely use double underscore prefixes and we document that anything starting with a single underscore is private. So I think we could perhaps align with the rest of the library and use a single underscore prefix here.

Reply:
Ok! Thanks!